cocotb/share/makefiles/Makefile.pylib.Darwin

```diff
@@ -35,24 +35,3 @@ ifneq ($(COCOTB_PYTHON_DEBUG),)
 else
 	PYTHON_BIN?=python
 endif
-
-NATIVE_ARCH=$(shell uname -m)
-ARCH?=$(NATIVE_ARCH)
-
-PYTHON_LIBDIR:=$(shell $(PYTHON_BIN) -c 'from __future__ import print_function; from distutils import sysconfig; print(sysconfig.get_config_var("LIBDIR"))')
-PYTHON_DYNLIBDIR:=$(shell $(PYTHON_BIN) -c 'from __future__ import print_function; from distutils import sysconfig; print(sysconfig.get_config_var("DESTSHARED"))')
-
-# Nasty hack but there's no way in Python properly get the directories for 32-bit Python on a 64-bit system
-# TODO: Under OSX we can use "export VERSIONER_PYTHON_PREFER_32_BIT=yes", no Linux equivalent though
-ifneq ($(NATIVE_ARCH),$(ARCH))
-	PYTHON_LIBDIR:=$(subst 64,,$(PYTHON_LIBDIR))
-	PYTHON_DYNLIBDIR:=$(subst 64,,$(PYTHON_DYNLIBDIR))
-endif
-
-PYLIBS:=$(shell $(PYTHON_BIN) -c 'from __future__ import print_function; from distutils import sysconfig; import os; print("-l"+os.path.splitext(sysconfig.get_config_var("LIBRARY"))[0][3:])')
-
-PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'from __future__ import print_function; import distutils.sysconfig; print(distutils.sysconfig.get_python_inc())')
-
-# Some Python installations can return a static Python library (libpython*.a) even if a shared Python library (libpython*.so) is
-# also present in a Python installation library path. The Python replace function is harmless for a Python dynamic installation
-PYTHON_DYN_LIB:=$(shell $(PYTHON_BIN) -c 'from __future__ import print_function; from distutils import sysconfig; print(sysconfig.get_config_var("LDLIBRARY").replace(".a", ".so"))')
```

Comment:
Is this still handled anywhere? Can we test this somehow?

Reply:
I have removed all of this.

Reply:
Does this mean that static python libraries (#1182) are broken again?

Reply:
Should not. It works slightly different now (hopefully better) and is independent what extension will be returned. 

```python
ld_library = sysconfig.get_config_var("LDLIBRARY")
python_lib_link = os.path.splitext(ld_library)[0][3:]
python_lib = "lib" + python_lib_link + "." + _get_lib_ext_name()
```

PS. I guess we can avoid this all together (as discussed above). Removing `PYTHON_SO_LIB`.